server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -64,6 +70,49 @@ public void upgradeMetadata(ServerContext context) {
     addHostingGoalToUserTables(context);
     deleteExternalCompactionFinalStates(context);
     deleteExternalCompactions(context);
+    removeCompactColumns(context);
+  }
+
+  private void removeCompactColumns(ServerContext context) {
+
+    final Text COMPACT_QUAL = new Text("compact");
+
+    try (var scanner = context.createScanner(MetadataTable.NAME);
```

Comment:
Should this use an isolated scanner?

Reply:
This method is effectively a copy of `deleteExternalCompactionFinalStates` and that's not being done there...